packages/pedantic/lint.js

```diff
@@ -1,11 +1,11 @@
-const format = require('./lib');
+import format from './lib.js';
 
-exports.command = '$0 <patterns..>';
+export const command = '$0 <patterns..>';
 
-exports.describe = 'Lint files use ESLint and prettier';
+export const describe = 'Lint files use ESLint and prettier';
 
-exports.builder = (_) =>
-  _.option('fix', {
+export function builder(_) {
```

Comment:
what's up with this argument being an underscore?

Reply:
just means unused